explainaboard/processors/conditional_generation.py

```diff
@@ -173,15 +173,32 @@ def default_metrics(
         cls, level='example', source_language=None, target_language=None
     ) -> list[MetricConfig]:
         eaas_defaults = cls._get_default_eaas_strs()
-        defaults: dict[str, list[MetricConfig]] = {
-            'example': [
-                EaaSMetricConfig(
-                    name=x,
-                    source_language=source_language,
-                    target_language=target_language,
+        full_metrics_human = [
```

Comment:
I think these should probably not be included in `default_metrics`, because these won't be calculated by default every time we use ExplainaBoard, right? Rather, they should be added to `full_metric_list()`

Reply:
I think you're right!